docusaurus/docs/operate/morse_migration/6_claiming_account.md

```diff
@@ -69,9 +69,18 @@ pocket accounts import-raw <priv-key-hex>
 pocketd keys add <your_shannon_key_name>
 ```
 
-### 3. Fund your Shannon account
+### 3. Create your onchain Shannon account
 
-You need to make sure the public key exists onchain and has funding to send the claim transaction.
+If you're using a newly generated key/account, then you will need to use one of the (network-specific) community faucets to trigger onchain account creation.
```

Comment:
Can you also add empty links with "coming soon" in the dev tools section: https://dev.poktroll.com/category/explorers-faucets-wallets-and-more

Reply:
I'm not sure I understand the ask here. :thinking:

Reply:
In `docusaurus/docs/tools/tools/shannon_beta.md`

```diff
+ - 🚰 Comming Soon: MACT Faucet
 - 🪙 [StakeNodes' Faucet](https://faucet.beta.testnet.pokt.network/)
 - 🗺️ [StakeNodes' Explorer](https://explorer.pocket.network/pocket-beta)
 - 🗺️ [Soothe's Explorer](https://shannon-beta.trustsoothe.io)
 - 👨‍💻 [Soothe's GraphQL Playground](https://shannon-beta-api.trustsoothe.io/)
```

Find an appropriate place to:
- Add a `; (TODO(@bryanchriswhite))`
- Link to the MACT docs

Repeat fr Alpha & MainNet.